### PR TITLE
Rewrite bootstrap.sh in /bin/sh

### LINKED ISSUE #191: Linux build instruction my be incorrect
This friendly comment popped up Twitter. Suggests there might be an issue with updated Linux build instructions. Do you think you can take a look @nschonni?

https://twitter.com/jfhbrook/status/776807851851251712

### PR DESCRIPTION
**Pull request by saper**
Fixes #191

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+clone_unless_present() {
+	[ -d "${1?clone_unless_present: need directory}" ] && return 0
+	git clone "${2?clone_unless_present: need URL}" "${1}"
+}
+
+clone_unless_present ${SASS_LIBSASS_PATH:=../libsass} https://github.com/sass/libsass &&
+clone_unless_present ${SASS_SPEC_PATH:=../sass-spec} https://github.com/sass/sass-spec
+
+export SASS_LIBSASS_PATH SASS_SPEC_PATH
```
Comment (nschonni) [inline bootstrap.sh]: I couldn't find a way to properly export enviornment variables from a bash shell. Is this supposed to break out of the subshell?

Comment (saper) [inline bootstrap.sh]: You can't. You need to source the script, that's why I have modified the instructions and the CI build script.

**Comment by jfhbrook** [comment]
I gave this a spin. [I was able to get it to work](https://github.com/jfhbrook/sass-tests/blob/master/Dockerfile.run-bootstrap-saper-branch), so in that sense mission accomplished. I wish the directions included `make install`, and I had some issues sourcing this script in /bin/sh, which is what docker's RUN command uses by default.

**Comment by nschonni** [comment]
LGTM, but it would be good to do a quick test on OSX since I just noticed we don't have that in our Travis matrix right now

**Comment by xzyfer** [comment]
I'm out of the loop here. @saper @nschonni feel free to make the call on this and :ship:

**Comment by marcenuc** [comment]
How can I help to get this merged?

Current unix build instructions are incomplete, so a lot of time is wasted trying to figure out what is wrong.

**Comment by saper** [comment]
@marcenuc :tada: :)

**Pull request merged**